platform/core/README.md

```diff
@@ -1,8 +1,23 @@
-# Rust implementations of polyPod platform core logic
+# PolyPod platform core
 
-The code is written against Rust 1.58.
+Logic shared across polyPod platforms - ios, android, wasm. 
 
-All you need is _cargo_, then run:
+## Requirements
+- [Rust toolchain](https://www.rust-lang.org/tools/install). Core is built with Rust 1.58.1.
 
-    cargo test
+## Building
 
+- When setting up the project for the first time or switching between branches, the flatbuffers models have to be generated:
+
+        make generate_flatbuffers
+    After models are generated, any changes to flatbuffers schema will trigger automatic update of the models when kernel is built.
```

Comment:
Still has "kernel" here -> There's also a bunch more occurrences below, but I won't point them all out individually since it's easy enough to search for them :D